static/src/js/util/request/colorMapRequest.js

```diff
@@ -0,0 +1,14 @@
+import Request from './request'
+import { getEnvironmentConfig } from '../../../../../sharedUtils/config'
+
+export default class ColorMapRequest extends Request {
+  constructor(authToken, earthdataEnvironment) {
+    super(getEnvironmentConfig().apiHost, earthdataEnvironment)
+    this.authenticated = true
```

Comment:
I don't think this constructor is correct, you don't have to be authenticated to retrieve colormaps. There is no authorizer in front of the getColorMap lambda and the lambda code does not look for any user header information.

You just need `this.lambda = true`, can remove `this.authenticated = true` and `this.authToken = authToken`